README.md

```diff
@@ -7,6 +7,7 @@ Jackdaw is a Clojure library for the Apache Kafka distributed streaming platform
 
 To use the latest release, add the following to your project:
 
+[![CircleCI](https://circleci.com/gh/FundingCircle/jackdaw.svg?style=svg)](https://circleci.com/gh/FundingCircle/jackdaw)
```

Comment:
The list of badges looks a bit weird when prefixed with the sentence "To use the latest release, add the following to your project". Maybe we should split out the clojars one from the others?

<img width="634" alt="screen shot 2019-02-25 at 9 24 43 pm" src="https://user-images.githubusercontent.com/48030/53369738-c4459f80-3943-11e9-8acb-58c95896ee51.png">

Reply:
Yes that's a good point, well I just suggest to move them all on top and remove the extra docs, I think Clojure devs can recognize the clojars badge anyway we don't really need to specify it